misc.php

```diff
@@ -79,3 +79,19 @@ function action_wpcom_vip_verify_string() {
 		}
 	}
 }, 10, 1 );
+
+/**
+ * Hooks pre_ping to stop any pinging from happening,
+ * unless `WPCOM_VIP_DO_PINGS` is set to `true`.
+ *
+ * @param array $post_links The URLs to be pinged
+ */
+function wpcom_vip_pre_ping( $post_links ) {
+	if ( ! defined( 'WPCOM_VIP_DO_PINGS' ) || true !== WPCOM_VIP_DO_PINGS ) {
+		return;
```

Comment:
Addressed in https://github.com/Automattic/vip-go-mu-plugins/pull/269/commits/0f66d853a3546c79118b56fa27f4bdbd330f459a